src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/generated.py

```diff
@@ -45,23 +45,24 @@
 
 # VM
 factory = lambda _: get_mgmt_service_client(VMClient).vm
-cli_command('vm create', VmOperations.create_or_update, factory, transform=DeploymentOutputLongRunningOperation('Starting vm create'))
+cli_command('vm create', VmOperations.create_or_update, factory, transform=DeploymentOutputLongRunningOperation('Starting vm create'),
+            simple_output_query="{ResourceGroup:resourceGroup, PublicIpAddress:publicIpAddress, PrivateIpAddress:privateIpAddress, MacAddress:macAddress, FQDN:fqdn}")
 
 factory = lambda _: _compute_client_factory().virtual_machines
 
-cli_command('vm delete', VirtualMachinesOperations.delete, factory)
-cli_command('vm deallocate', VirtualMachinesOperations.deallocate, factory)
-cli_command('vm generalize', VirtualMachinesOperations.generalize, factory)
-cli_command('vm show', VirtualMachinesOperations.get, factory, simple_output_query="{Name:name, ResourceGroup:resourceGroup, Location:location, VmSize:hardwareProfile.vmSize, OsType: storageProfile.osDisk.osType, Urn: join(':', [storageProfile.imageReference.publisher, storageProfile.imageReference.offer, storageProfile.imageReference.sku, storageProfile.imageReference.version])}")
-cli_command('vm list-vm-resize-options', VirtualMachinesOperations.list_available_sizes, factory)
-cli_command('vm stop', VirtualMachinesOperations.power_off, factory)
-cli_command('vm restart', VirtualMachinesOperations.restart, factory)
-cli_command('vm start', VirtualMachinesOperations.start, factory)
-cli_command('vm redeploy', VirtualMachinesOperations.redeploy, factory)
-cli_command('vm list-ip-addresses', list_ip_addresses)
-cli_command('vm list', list_vm, simple_output_query="[*].{Name: name, ResourceGroup: resourceGroup, Location: location, VmSize: hardwareProfile.vmSize, Urn: join(':', [storageProfile.imageReference.publisher, storageProfile.imageReference.offer, storageProfile.imageReference.sku, storageProfile.imageReference.version])} | sort_by(@, &Name)")
-cli_command('vm resize', resize_vm)
-cli_command('vm capture', capture_vm)
+cli_command('vm delete', VirtualMachinesOperations.delete, factory) # no output
```

Comment:
you probably don't this comment ` # no output`, as it is not being applied on all cases